demos/shadow/shadow_demo_main/demo_config.h

```diff
@@ -153,18 +153,30 @@
  * symbol.
  */
 #include "core_mqtt.h"
-#define MQTT_LIB             "core-mqtt@" MQTT_LIBRARY_VERSION
+#define MQTT_LIB              "core-mqtt@" MQTT_LIBRARY_VERSION
 
 /**
  * @brief Predefined thing name.
  *
  * This is the example predefine thing name and could be compiled in ROM code.
  */
-#define THING_NAME           "testShadow"
+#define THING_NAME            "testShadow"
 
 /**
  * @brief The length of #THING_NAME.
  */
-#define THING_NAME_LENGTH    ( ( uint16_t ) ( sizeof( THING_NAME ) - 1 ) )
+#define THING_NAME_LENGTH     ( ( uint16_t ) ( sizeof( THING_NAME ) - 1 ) )
+
+/**
+ * @brief Predefined shadow name.
+ *
+ * Defaults to unnamed "Classic" shadow. Change to a custom string to use a named shadow.
+ */
+#define SHADOW_NAME           SHADOW_NAME_CLASSIC
```

Comment:
Please add a #ifndef check around it so that the CI will be able to configure the demo to work as a classic shadow demo vs named shadow demo.

Reply:
Done.